libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -62,7 +62,6 @@
 #define HAL_INS_NONE         0
 #define HAL_INS_MPU60XX_SPI  2
 #define HAL_INS_MPU60XX_I2C  3
-#define HAL_INS_HIL          4
```

Comment:
leave these in as deprecated, so we don't reuse

Reply:
Fixed